apps/api/Dockerfile

```
@@ -30,4 +30,4 @@ RUN adduser --system --uid 1001 nitro
 USER nitro
 COPY --from=installer /app .
 
-CMD pnpm srvx --prod .output/ --port 3001
\ No newline at end of file
+CMD pnpm srvx --prod .output/ --port 8080
```

Hadolint (2.14.0)

[warning] Use arguments JSON notation for CMD and ENTRYPOINT arguments

(DL3025)